third_party/move/move-core/types/src/vm_status.rs

```diff
@@ -305,10 +311,11 @@ impl fmt::Display for KeptVMStatus {
                 location,
                 function,
                 code_offset,
+                message,
             } => write!(
                 f,
-                "EXECUTION_FAILURE at bytecode offset {} in function index {} in {}",
-                code_offset, function, location
+                "EXECUTION_FAILURE at bytecode offset {} in function index {} in {} with error message {}",
+                code_offset, function, location, message.clone().unwrap_or("".to_string())
```

Comment:
why cloning the message? 
why not just use "{:?}" to show "None"?

Reply:
updated to :?